src/components/Table/TableShared.part.tsx

```diff
@@ -104,20 +59,45 @@ export const StyledTableHeader = styled('th')<TableHeaderProps>`
   ${({ width }) => (width && `width: ${width}`) || ''};
 `;
 
-export function defaultHeaderCellRenderer<T>({ value, sorting, column }: TableCellRenderEvent<T>): React.ReactNode {
+export const getDefaultHeaderCellRenderer = ({ setState }: { setState: any }) => <T extends {}>({
```

Comment:
`({ setState }: { setState: any })` agrument doesn't look good here. Why not accept `IncreaseDecreaseProps` obviously here? 
```ts
getDefaultHeaderCellRenderer({ onIncrease, onDecrease })
```